### PR TITLE
feat: added support for new OCI regions: Milan, Stockholm, Abu Dhabi and Vinhedo

### LINKED ISSUE #440: Add support for new OCI regions Milan and Stockholm
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

2 new OCI regions have been launched, we need to add their OCIR urls so the secret can be created accordingly.

### Potential Terraform Configuration

```hcl
# Copy-paste any Terraform configurations for how the requested feature may be used. 
```

### PR DESCRIPTION
**Pull request by snafuz**
Close #440 

Signed-off-by: Andrea Marchesini <andrea.marchesini@oracle.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,11 +6,13 @@ provider.tf
 # .tfstate files
 *.tfstate
 *.tfstate.*
+.terraform.lock.hcl
```
Comment (hyder) [inline .gitignore:9]: Please remove.

File: variables.tf [hunk 2]
```diff
@@ -660,31 +660,35 @@ variable "ocir_urls" {
...
-    sa-saopaulo-1  = "gru.ocir.io"
-    sa-santiago-1  = "scl.ocir.io"
-    uk-cardiff-1   = "cwl.ocir.io"
-    uk-london-1    = "lhr.ocir.io"
-    us-ashburn-1   = "iad.ocir.io"
-    us-phoenix-1   = "phx.ocir.io"
-    us-sanjose-1   = "sjc.ocir.io"
+    ap-melbourne-1  = "mel.ocir.io"
+    ap-hyderabad-1  = "hyd.ocir.io"
+    ap-mumbai-1 	  = "bom.ocir.io"
```
Comment (hyder) [inline variables.tf:665]: Formatting required

**Review by hyder — CHANGES_REQUESTED** [review]
Minor changes required

File: variables.tf (hunk 2 above)
Comment (snafuz) [reply variables.tf]: replaced tabs with spaces  in rev.1

File: .gitignore (hunk 1 above)
Comment (snafuz) [reply .gitignore]: why this should be removed?

Comment (hyder) [reply .gitignore]: We discussed this earlier internally and we concluded that it was ok for examples but not required for the module itself.

Let's remove it for the purpose of this PR and discuss this again when everybody is back on deck.

**Review by hyder — APPROVED** [review]
approved

**Pull request merged**